packaging/docker/entrypoint.sh

```
@@ -34,4 +34,16 @@ fi
 mkdir -p /app/data /bookdrop /books
 chown "$USER_ID:$GROUP_ID" /app/data /bookdrop /books 2>/dev/null || true
 
+# Process _FILE environment variables for secrets
+for var in $(env | grep '_FILE=' | cut -d= -f1); do
+    file_path=$(eval echo \$$var)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)